src/react_native/permissions.cljs

```diff
@@ -95,3 +95,7 @@
   []
   (-> (checkNotifications)
       (promesa/then notification-permissions->notification-permission-statuses)))
+
+(defn open-notification-settings
+  []
+  (openSettings "notifications"))
```

Comment:
Here we're creating a function that should open the OS system settings screen for the app, and if possible it will display the notification related settings for the app (from the OS System pov, not inside Status app).